app/managers.py

```diff
@@ -0,0 +1,50 @@
+import sqlite3
+from typing import List
+
+from app.models import Actor
```

Comment:
Order your imports according to PEP-8, `typing` is a standard library import, `sqlite3` is a third-party import. Delimit different groups of imports with an empty line.

Reply:
Thank you, I've changed it